styles/mainSlider.scss

```diff
@@ -0,0 +1,52 @@
+.main-slider {
+  position: relative;
+
+  .main-slider-arrow {
+    display: flex;
+    flex-flow: column nowrap;
+
+    height: 100%;
+    width: 10%;
+    width: 10vw;
+
+    align-items: center;
+    justify-content: center;
+    z-index: 3;
+
+    font-size: 42px;
+    font-size: 5vw;
+    color: transparent;
+    background-color: transparent;
+    cursor: pointer;
+
+
+    &:hover {
+      color: white;
+      background-color: grey;
```

Comment:
do we want to use grey or maybe put some other shade into variable for consistency?